htdocs/class/userutility.php

```diff
@@ -100,13 +100,13 @@ public static function validate()
                 $user = $args[0];
                 break;
             case 2:
-                list($uname, $email) = $args;
+                [$uname, $email] = $args;
```

Comment:
[] in list unpacking is only allowed since PHP 7.1

Please revert the 5 instances of this in this file